conf/logback.xml

```diff
@@ -43,27 +43,24 @@
         <appender-ref ref="STDOUT_IGNORE_NETTY" />
     </logger>
 
-    <logger name="com.ning.http.client" level="WARN"/>
-    <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.core.netty.utils.ServerCookieDecoder" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.request.NettyRequestSender" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.handler.HttpHandler" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.ChannelManager" level="WARN"/>
-    <logger name="org.asynchttpclient.netty" level="WARN"/>
+    <logger name="com.google.inject" level="WARN"/>
 
-    <logger name="uk.gov" level="INFO"/>
+    <logger name="uk.gov" level="WARN"/>
 
-    <logger name="application" level="DEBUG"/>
+    <logger name="application" level="WARN"/>
 
-    <logger name="com.google.inject" level="INFO"/>
-
-    <logger name="connector" level="TRACE">
+    <logger name="connector" level="WARN">
         <appender-ref ref="STDOUT"/>
     </logger>
 
-    <root level="DEBUG">
+    <logger name="com.codahale.metrics.graphite.GraphiteReporter" level="OFF" />
```

Comment:
Re all of these set as WARN:
According to acceptance criteria should this not remain as INFO ?
'Set the logger levels in logback.xml to the same as what we had set for app-config-production'

Reply:
Yes, should be INFO, i've copied a section from https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml and then forgotten to change back to INFO